src/Microsoft.Diagnostics.EventFlow.Outputs.ElasticSearch/ElasticSearchOutput.cs

```diff
@@ -164,19 +164,6 @@ private IEnumerable<IBulkOperation> GetCreateOperationsForEvent(EventData eventD
                 }
             }
 
-            if (eventData.TryGetMetadata(DependencyData.DependencyMetadataKind, out metadataSet))
```

Comment:
What's the bug? Is CreateDependencyOperation() still needed then?

Reply:
Duplicate code, see https://github.com/Azure/diagnostics-eventflow/issues/385